### ISSUE TITLE
The accuracy and loss of ENAS algorithm are not convergent. Is there any appropriate parameter setting for the optimizer

### ISSUE THREAD

**Issue by Drxdx** [microsoft/nni#5632]
**Describe the issue**:
Tensorboard shows:The accuracy and loss of ENAS algorithm are not convergent. Is there any appropriate parameter setting for the optimizer.
**Environment**:
- NNI version:2.10
- Training service (local|remote|pai|aml|etc):remote
- Client OS:Ubuntu

**How to reproduce it?**:

**Comment by matluster** [comment]
Lower learning rate might be helpful.

**Comment by matluster** [comment]
Lower learning rate might be helpful.
